patches/0004-Use-crypto-backends.patch

```diff
@@ -379,19 +380,162 @@ index 33942467784ad3..0282ffa9fa23c8 100644
  	"crypto/internal/cryptotest"
  	fipsaes "crypto/internal/fips140/aes"
  	"encoding/hex"
+diff --git a/src/crypto/cipher/gcm.go b/src/crypto/cipher/gcm.go
+index 5580f96d55a0fb..cc759af19b116b 100644
+--- a/src/crypto/cipher/gcm.go
++++ b/src/crypto/cipher/gcm.go
+@@ -93,7 +93,14 @@ func newGCM(cipher Block, nonceSize, tagSize int) (AEAD, error) {
+ func NewGCMWithRandomNonce(cipher Block) (AEAD, error) {
+ 	c, ok := cipher.(*aes.Block)
+ 	if !ok {
+-		return nil, errors.New("cipher: NewGCMWithRandomNonce requires aes.Block")
```

Comment:
We still need to return this error if the cipher doesn't implement `cipher` nor the gcm implements `randomNonceSealer.`